packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -3296,6 +3298,176 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /producerKeychains:
+    get:
+      tags:
+        - producerKeychains
+      summary: List Producer Keychains
+      description: Retrieve a list of Producer Keychains based on filters
+      operationId: getProducerKeychains
+      parameters:
+        - in: query
+          name: name
```

Comment:
we probably have a visibility conflict here: is it possible for a non-owner tenant to filter by name even though it can not see the field in return?

Reply:
We have the same in `getClients`, which is already deployed in production 😅

Reply:
Uh-oh, does it make sense to show the name at this point?